insights/parsers/numeric_user_group_name.py

```diff
@@ -0,0 +1,62 @@
+"""
+NumericUserGroupName - command ``/usr/bin/grep -c '^[[:digit:]]' /etc/passwd /etc/group``
+=========================================================================================
+
+This parser reads the output of ``/usr/bin/grep -c '^[[:digit:]]' /etc/passwd /etc/group``
+and detects whether it is 0 or nonzero.
+
+The grep command returns the number of matches. It matches for user and group names that
```

Comment:
It actually does not check return value, I am not sure if it is obvious, that the command prints the statistics and we have no way how to check return values.